apps/cli/cli_common.c

```diff
@@ -774,6 +774,67 @@ cli_set_mode(clixon_handle h,
     return retval;
 }
 
+/**
+ * @brief Runs a Python script in a new process
+ *
+ * @param h Clicon handle
+ * @param cvv CLIgen variable vector
+ * @param argv function arguments
+ *
+ * The function creates a new child process in which the Python script is launched.
+ * In the child process, before launching the script, environment variables are set
+ * taken from cvv.
+ *
+ * The function returns the exit code of the Python script, or a value of (-1) on error.
+ *
+ * @return Returns 0 on success, -1 in case of error
+ */
+int cli_start_python3(clixon_handle h, cvec *cvv, cvec *argv) {
+    int pid = 0;
+    int status = 0;
+    char *script_path = NULL;
+    char *runner = "python3";
+    struct passwd *pw = NULL;
+
+    if (cvec_len(argv) > 1){
+        fprintf(stderr,"A lot of arguments");
```

Comment:
Please look at CONTRIBUTING.md regarding style. Use clixon_err for errors and preferably a done: clasue instead of direct returns.